app/api/helpers/payment.py

```diff
@@ -66,6 +72,27 @@ def get_credentials(event=None):
             else:
                 return None
 
+    @staticmethod
+    def get_event_organizer_credentials_from_stripe(stripe_auth_code):
+        """
+        Uses the stripe_auth_code to get the other credentials for the event organizer's stripe account
+        :param stripe_auth_code: stripe authorization code
+        :return: response from stripe
+        """
+        credentials = StripePaymentsManager.get_credentials()
+
+        if not credentials:
+            raise Exception('Stripe is incorrectly configured')
```

Comment:
You should probably add a more meaningful error message, say that the credentials were not found.

Reply:
I agree